mir-ci/mir_ci/program.py

```diff
@@ -27,23 +34,33 @@ def format_output(name: str, output: str) -> str:
 class ProgramError(RuntimeError):
     pass
 
-class Program:
-    def __init__(self, command: Command, env: Dict[str, str] = {}):
+class Program(Benchmarkable):
+    def __init__(
+            self,
+            command: Command,
+            env: Dict[str, str] = {},
+            with_systemd_run: bool = False):
```

Comment:
IMO this can simply be the default, there's no reason to have some Programs under a cgroup, but not others, is there?

Reply:
My reason was for snaps, but actually the regular cgroup directory would still be made for snaps anyway it seems

Reply:
If we undo this, it looks like we can _really_ simplify things. I will do that!